examples/ola-streaming-client.cpp

```diff
@@ -41,6 +41,12 @@ DEFINE_s_string(dmx, d, "", "Comma separated DMX values to send, e.g. "
 DEFINE_s_uint32(universe, u, 1, "The universe to send data for");
 DEFINE_uint8(priority, ola::dmx::SOURCE_PRIORITY_DEFAULT,
              "The source priority to send data at");
+DEFINE_s_default_bool(universe_from_source, s, false,
```

Comment:
I'm wondering if this would be better called --universe-from-stdin or something. Source doesn't jump out as obvious to me.

Reply:
Makes sense, I'll go swap those out.